libs/blocks/review/utils/utils.js

```diff
@@ -9,4 +9,14 @@ const isKeyboardNavigation = (ev) => (ev.clientX === 0 && ev.clientY === 0)
     && ev.nativeEvent.webkitForce !== undefined
     && ev.nativeEvent.webkitForce === 0);
 
-export { addToAverage, isKeyboardNavigation };
+const checkPostUrl = (postUrl, env) => {
+  const url = new URL(postUrl);
```

Comment:
maybe we need some try/catch here to ensure we catch errors when URLs are not valid fomat.